Formula/voltdb.rb

```diff
@@ -14,11 +14,14 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project as a pull-request and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!